tests/functional/aws-node-sdk/test/versioning/bucketDelete.js

```diff
@@ -25,72 +27,70 @@ describe('aws-node-sdk test delete bucket', () => {
         const s3 = bucketUtil.s3;
 
         // setup test
-        beforeEach(done => {
-            async.waterfall([
-                next => s3.createBucket({ Bucket: bucketName },
-                    err => next(err)),
-                next => s3.putBucketVersioning({
-                    Bucket: bucketName,
-                    VersioningConfiguration: {
-                        Status: 'Enabled',
-                    },
-                }, err => next(err)),
-            ], done);
+        beforeEach(async () => {
+            await s3.send(new CreateBucketCommand({ Bucket: bucketName }));
+            await s3.send(new PutBucketVersioningCommand({
+                Bucket: bucketName,
+                VersioningConfiguration: {
+                    Status: 'Enabled',
+                },
+            }));
         });
 
         // empty and delete bucket after testing if bucket exists
-        afterEach(done => {
+         afterEach(done => {
```

Comment:
```suggestion
        afterEach(done => {
```

indentation ? Weird that the CI pass 😬